twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/data/datasource/usertweets/UserTweetsDataSourceFactory.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.vnicius.twitterclone.data.datasource.usertweets
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.paging.DataSource
+import io.github.vnicius.twitterclone.data.repository.user.UserRepository
+import twitter4j.Status
+
+class UserTweetsDataSourceFactory(
+    private val userId: Long,
+    private val itemsCount: Int,
```

Comment:
If the `itemsCount` means the page size I think it makes more sense to use the name `pageSize`.